docs/devices/915005997501.md

```diff
@@ -19,7 +19,7 @@ pageClass: device-page
 | Vendor  | Philips  |
 | Description | Hue Bluetooth white & color ambiance ceiling lamp Infuse large |
 | Exposes | light (state, brightness, color_temp, color_temp_startup, color_xy, color_hs), effect, linkquality |
-| Picture | ![Philips 915005997501](https://www.zigbee2mqtt.io/images/devices/915005997501.jpg) |
+| Picture | ![Philips 915005997501](https://www.zigbee2mqtt.io/images/devices/915005997301.jpg) |
```

Comment:
This part is generated, so please update the picture file instead (`915005997501.jpg`)